pkg/admissioncontroller/webhooks/auth/seed/graph/eventhandler_managedseed.go

```diff
@@ -116,6 +117,8 @@ func (g *graph) handleManagedSeedCreateOrUpdate(ctx context.Context, managedSeed
 	} else if seed.Status.ClientCertificateExpirationTimestamp != nil && seed.Status.ClientCertificateExpirationTimestamp.UTC().Before(time.Now().UTC()) {
 		// Seed is registered but the client certificate expiration timestamp is expired.
 		allowBootstrap = true
+	} else if managedSeed.Annotations[v1beta1constants.GardenerOperation] == v1beta1constants.GardenerOperationRenewKubeconfig {
+		allowBootstrap = true
```

Comment:
Did you test it this time/does this work? It seems to me that we delete the operation annotation in https://github.com/gardener/gardener/blob/debfabc71d52d1848febe5174274e1bc819a3668/pkg/gardenlet/controller/managedseed/managedseed_actuator.go#L719-L723 which is before the gardenlet is redeployed for bootstrapping again. This would mean that once gardenlet comes up it will fail to bootstrap again. Am I missing something?

Reply:
I tested it using garden-setup with a manually deployed ManagedSeed resource, and the kubeconfig secret was re-created properly when I annotated the managedseed. But now that you say this, I wonder why this works too 🤔

Reply:
Maybe it would be better to check for non-existence of the kubeconfig secret there? Just to be sure?
Or move the deletion of the 'renew-kubeconfig' secret into the bootstrapping logic, but I'm not sure of that one ...